# 场地事件

Header header

# 补血点信息
rm_referee_controller/EventRestoration restoration
# 能量机关信息
rm_referee_controller/EventPowerRune powerRune
# 高地信息
rm_referee_controller/EventElevated elevated
# 基地虚拟护盾是否还有血量
bool isVirtualShieldActive
# 前哨站是否存活
bool isOutpostActive